charts/pega/charts/hazelcast/README.md

```diff
@@ -1,4 +1,4 @@
-# Clustering Service Deployment (Draft)
+# Clustering Service Deployment
 
 Clustering Service is Pega's own docker image containing the Hazelcast Clustering Service image inside it. 
```

Comment:
small tweaks>>
The Hazelcast clustering service is implemented using a Pega-provided Docker image that contains the Hazelcast clustering service image, which you must make available in your deployment Docker image repository.
This Hazelcast subchart defines the additional deployment parameters the clustering service uses to define dynamic features for the client-server deployment model using Hazelcast.